tests/unit/ui/components/TestRating.js

```diff
@@ -180,22 +193,35 @@ describe(__filename, () => {
 
   it('renders an accessible description for null stars', () => {
     const root = render({ rating: null });
-    expect(findDOMNode(root).textContent).toEqual('Click to rate this add-on');
+
+    [1, 2, 3, 4, 5].forEach((rating) => {
+      expect(root.ratingElements[rating].title).toEqual(`Rate this add-on ${rating} out of 5.`);
+    });
   });
 
   it('renders an accessible description for 0 stars', () => {
     const root = render({ rating: 0 });
-    expect(findDOMNode(root).textContent).toEqual('Click to rate this add-on');
+
+    [1, 2, 3, 4, 5].forEach((rating) => {
+      expect(root.ratingElements[rating].title).toEqual(`Rate this add-on ${rating} out of 5.`);
+    });
   });
 
-  it('renders "no ratings" if no rating and readOnly', () => {
+  it('renders "This add-on has not been rated yet." if no rating and readOnly', () => {
     const root = render({ rating: null, readOnly: true });
-    expect(findDOMNode(root).textContent).toEqual('This add-on has no ratings.');
+
+    expect(findDOMNode(root).title).toContain('This add-on has not been rated yet.');
   });
 
   it('renders an accessible description for ratings', () => {
     const root = render({ rating: 2 });
-    expect(findDOMNode(root).textContent).toEqual('Rated 2 out of 5');
+
+    [1, 2, 3, 4, 5].forEach((rating) => {
+      expect(root.ratingElements[rating].title).toEqual(`Update your rating to ${rating} out of 5.`);
```

Comment:
I think you should be testing to make sure these are not set when `readOnly` is set and I'm not sure I see that test anywhere.

Reply:
I went ahead and added another test for that case, found [here](https://github.com/mozilla/addons-frontend/pull/4693/files#diff-7ac6d4cc535e144232c13e382e52eeecR224)!